src/amo/components/AddonMeta/index.js

```diff
@@ -70,7 +86,14 @@ export class AddonMetaBase extends React.Component<Props> {
                   styleSize="small"
                 />
               ),
-              title: i18n.gettext('Overall Rating'),
+              title: (
```

Comment:
I believe the rating stars (at line 82, above) are also meant to be turned into a link.